optimum/habana/checkpoint_utils.py

```diff
@@ -92,7 +92,7 @@ def model_on_meta(config):
     """
     Checks if load the model to meta.
     """
-    return config.model_type in ["bloom", "llama", "falcon"]
+    return config.model_type in ["bloom", "llama", "falcon", "mixtral"]
 
```

Comment:
@skaulintel  what is this change? Do we need to add Mistral as well?

Reply:
probably, @libinta can you confirm?

Reply:
actually not need for mistral @jiminha